frontend/src/utils/index.js

```diff
@@ -10,9 +10,7 @@ export function isAdmin() {
   return true;
 }
 
-export function isLoggedIn() {
-  return true;
-}
+export const isLoggedIn = () => localStorage.hasOwnProperty("AUTH_TOKEN");
```

Comment:
Do you plan to move this elsewhere in the future?

Reply:
hmm would something like the `api` module be a better place for this?

Reply:
Oh no, I meant more that you're storing the auth token in local storage (as in do you plan to use a more secure method of storing auth state at some point)

Reply:
Oh right, yeah we're planning to move it to cookies instead

Reply:
(This is irrelevant to this PR, and hence it's a comment rather than a change request; sounds good)